packages/firestore/src/protos/firestore_proto_api.d.ts

```diff
@@ -180,7 +180,7 @@ export declare namespace firestoreV1ApiClientInterfaces {
     name?: string;
     fields?: ApiClientObjectMap<Value>;
     createTime?: string;
-    updateTime?: string;
+    updateTime?: string | { seconds?: string | number; nanos?: number };
```

Comment:
Is this generated or hand edited?

If hand edited, would it be worthwhile to define the timestamp type in here and reuse it for all these declarations?

Then we could just call it `Timestamp` and refer to it as `api.Timestamp`.

Reply:
This used to be generated, but has been hand edited many times since then. As such, I added `api.Timestamp`.